Library/Homebrew/caveats.rb

```diff
@@ -117,10 +128,20 @@ def function_completion_caveats(shell)
           #{root_dir}/etc/bash_completion.d
       EOS
     when :zsh
-      <<~EOS
+      site_functions = root_dir/"share/zsh/site-functions"
+      zsh_caveats = +<<~EOS
         zsh #{installed.join(" and ")} have been installed to:
-          #{root_dir}/share/zsh/site-functions
+          #{site_functions}
       EOS
+      unless PATH.new(ENV["HOMEBREW_FPATH2"]).to_a.include?(site_functions.to_s)
```

Comment:
typo? HOMEBREW_FPATH2?

Reply:
@yschimke good catch! Changed it during testing. Thanks!